packages/lib-classifier/src/plugins/tasks/DrawingTask/components/DrawingTask.stories.js

```diff
@@ -1,81 +1,8 @@
 import { withKnobs, boolean, select } from '@storybook/addon-knobs'
 import asyncStates from '@zooniverse/async-states'
 import { storiesOf } from '@storybook/react'
-import zooTheme from '@zooniverse/grommet-theme'
```

Comment:
Import of the theme should remain as its used throughout lines 89-125

Reply:
Done!